### ISSUE TITLE
Incremental reporters

### ISSUE THREAD

**Issue by tsenart** [tsenart/vegeta#82]
With the previous work on making most of the data pipeline stream results as they come, it would be great to have a some new reporters which incrementally reflect new incoming data.

**Comment by mickey** [comment]
TLDR; https://asciinema.org/a/16198

Hey @tsenart [here](https://github.com/mickey/vegeta/commit/64412d7a1a62d098140bd3df8b9529951894e6a6) is a basic prototype for an incremental reporter that prints the latency of every requests stdout.
In my video I pipe this to [feedgnuplot](https://github.com/dkogan/feedgnuplot) to have a real time plot :D

Looking at [only the first 4 characters](https://github.com/tsenart/vegeta/blob/master/report.go#L33) of the reporter name might be a problem if adding more.

If you're interested I'd happily submit a pull request. Otherwise, what kind of reporter do you have in mind?

**Comment by tsenart** [comment]
Hey @mickey,

Thanks for your interest in this issue and for showing me `feedgnuplot`! I didn't know about it.
Since we now have a `dump` command which can be used in a pipeline with tools like `feedgnuplot`, I believe this issue can be closed. With that said, I don't think your change set is required to achieve what you want. Mind, though, that the latency is in nanoseconds so you'd have to convert it to milliseconds first. Anyways, here's an example (beware the possible output buffering done by awk which is platform dependent).

```
echo "GET http://:6060/" | vegeta attack -duration=60s -rate=200 | tee results.bin \
| vegeta dump -dumper="json" | jq '.latency' | awk '{print $1/1e6}'
```

**Comment by mickey** [comment]
Hey @tsenart thanks for your feedback!
I agree it doesn't make sense  with the use of the `dump` command.
This is perfect, thanks!
